app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -119,12 +119,12 @@ class ProfileEditActivityTest {
   @Test
   fun testProfileEdit_updateName_checkNewNameDisplayed() {
     profileManagementController.updateName(
-      ProfileId.newBuilder().setInternalId(1).build(),
+      profileId = ProfileId.newBuilder().setInternalId(1).build(),
```

Comment:
This change is not required as the value itself if self explanatory.

Reply:
Done